pfb-analysis/connectivity/census_blocks.sql

```diff
@@ -4,17 +4,117 @@
 -- code to be run on table that has
 -- been imported directly from US Census
 -- blkpophu file
+-- :nb_output_srid psql var must be set before running this script,
+--      e.g. psql -v nb_output_srid=2249 -f census_blocks.sql
```

Comment:
There's a discussion started in #152 about what the default SRID should be. We'll want to update the comments here and in the remainder of this PR with the default SRID chosen, so that we're consistent across files.